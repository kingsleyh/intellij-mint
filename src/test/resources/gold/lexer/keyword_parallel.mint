component Cool {

  fun render : Html {
    </div>
  }

  fun go() : Void {
    parallel {
      something()
    }
  }

}