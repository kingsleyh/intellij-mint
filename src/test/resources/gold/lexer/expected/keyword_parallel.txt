component ('component')
WHITE_SPACE (' ')
CAPITALISED_IDENT ('Cool')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n\n  ')
fun ('fun')
WHITE_SPACE (' ')
LOWER_IDENT ('render')
WHITE_SPACE (' ')
: (':')
WHITE_SPACE (' ')
CAPITALISED_IDENT ('Html')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n    ')
< ('<')
/ ('/')
LOWER_IDENT ('div')
> ('>')
WHITE_SPACE ('\n  ')
} ('}')
WHITE_SPACE ('\n\n  ')
fun ('fun')
WHITE_SPACE (' ')
LOWER_IDENT ('go')
( ('(')
) (')')
WHITE_SPACE (' ')
: (':')
WHITE_SPACE (' ')
CAPITALISED_IDENT ('Void')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n    ')
parallel ('parallel')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n      ')
LOWER_IDENT ('something')
( ('(')
) (')')
WHITE_SPACE ('\n    ')
} ('}')
WHITE_SPACE ('\n  ')
} ('}')
WHITE_SPACE ('\n\n')
} ('}')